socorro/cron/jobs/reprocessingjobs.py

```diff
@@ -30,10 +31,24 @@ class ReprocessingJobsApp(BaseCronApp):
     app_version = '0.1'
 
     def run(self, connection):
+        select_sql = """
+            SELECT crash_id FROM reprocessing_jobs LIMIT 10000
+        """
+        crash_ids = []
+        for crash_id, in execute_query_iter(connection, select_sql):
+            crash_ids.append(crash_id)
+
+        delete_sql = """
+            DELETE from reprocessing_jobs WHERE crash_id = %(crash_id)s
+        """
 
-        for crash_id, in execute_query_iter(connection, _reprocessing_sql):
+        for crash_id in crash_ids:
             self.queuing_connection_factory.save_raw_crash(
                 DotDict({'legacy_processing': 0}),
                 [],
                 crash_id
             )
+            execute_no_results(connection, delete_sql, {
+                'crash_id': crash_id
+            })
+            connection.commit()
```

Comment:
You're right. I forgot about that. If the `.execute()` fails, the `.commit()` will also fail. 

So I think it's best, and safe to leave it as is. 10,000 commits. Solid and safe. And rare too. And, best of all, if 10,000 is too much and it causes pain on the PG side, crontabber will simply try again 5 minutes later.